daffodil-lib/src/main/scala/org/apache/daffodil/schema/annotation/props/PropertyScoping.scala

```diff
@@ -131,6 +144,21 @@ trait FindPropertyMixin extends PropTypes {
           lr
         }
       }
+    // if this looks like it could be an expression, issue a warning
+    // but only if expressionAllowed is false
+    if(!expressionAllowed) {
+      propRes match {
+        case Found(v, _, _, _) => {
+          if(DPathUtil.isExpression(v)) {
```

Comment:
Formatting: should be `if (` not `if(`

Reply:
Made a fix-up commit before rebasing that resolves this.